tmeister/static/src/release-notes/release-notes-table/release-note-row.component.js

```diff
@@ -1,39 +1,61 @@
 import React, { useState } from 'react'
 import { md } from '../release-notes.helper.js'
+import { makeStyles } from '@material-ui/core/styles'
 import { TableRow, TableCell, TableBody, Paper, IconButton, Icon } from '@material-ui/core'
 import CreateEditReleaseNote from '../create-edit/create-edit.component.js'
 
+const useStyles = makeStyles(theme => ({
+  flex: {
+    display: 'flex'
+  }
+}))
+
 export default function ReleaseNoteRow (props) {
   const { note = {} } = props
+  const classes = useStyles()
   const { title, body, relatedToggles } = note
   const [inlinePreview, setInlinePreview] = useState(true)
   const [edit, setEdit] = useState(false)
+  const [showArrow, preview] = previewBody(body)
   return (
     <TableRow key={note.id}>
       <TableCell>
         {title}
       </TableCell>
       <TableCell>
-        <div>
-          {previewBody(body)}
+        <div className={classes.flex}>
+          <div>
+            {preview}
+          </div>
+          {
+            showArrow && (
+              <div>
+                <IconButton onClick={() => setInlinePreview(!inlinePreview)}>
+                  <Icon>
+                    { inlinePreview ? 'arrow_drop_down' : 'arrow_drop_up'}
+                  </Icon>
+                </IconButton>
+              </div>
+            )
+          }
```

Comment:
Only showing the arrow if it makes sense to show (if there is stuff to show/hide)